### DISCUSSION TITLE
Integration von VTC-Diensten

### DISCUSSION THREAD

**Discussion by fanie-mak** [e-netze/webgis-community#163] [General]
Prototypisch wurden VTC Dienste im Geoland-Viewer eingebunden 
https://www.geoland.at/webgisviewer/geoland/map/Geoland_Viewer/Geoland_VTC

Ziel:
Einbindung von VTC als Hintergrundkarte + Karte am Ausdruck (ohne Abfrage) --> Entwickeln einer gemeinsamen Vorgangsweise

**Comment by gis-eni** [comment]
Vielleicht beschreibst du auch ein wenig, wie du jetzt den VTC eingebunden hast? Ich hab euch ja mehrere Libraries angeboten (mapbox, maplibre, ...). Damit ich dann auch die Library verwende, dir für eure VTCs funktioniert.

Hintergrundkarte im Viewer wäre dann möglich (keine Abfragen).

**Zum Drucken:**
Ist leider nicht möglich. Ein Rendern der Tiles am Server ist schwierig. Bestehende Libraries beschränken sich oft nur auf Teile der Möglichkeiten (zB. kein Labeln, also keine Beschriftung, weil die richtige Platzierung dann schon etwas anspruchsvoller wird)

Man wird für VTCs also einen "Fallback" Dienst angeben müssen, der für das Rendern am Server verwendet wird. Das kann dann irgendein Kartendienst oder Raster Tiles Dienst sein, wie sie jetzt schon im WebGIS möglich sind.
zB: Client VTC Ortsplan => Server Basemap Raster Tiles
Oder so etwas wie der OpenMapTiles Server, der on the fly Tiles in WMS umwandeln kann (oder?)... 
Auf alle Fälle wird WebGIS keine serverseitiges Rendern von VTCs anbieten.
